### PR TITLE
Remove obsolete include and Xlib comment.

### PR DESCRIPTION
**Pull request by Airblader**

### CODE REVIEW

File: objects/key.c [hunk 1]
```diff
@@ -38,7 +38,7 @@
 #include "common/xutil.h"
 #include "xkb.h"
 
-/* XStringToKeysym() and XKeysymToString */
+/* XStringToKeysym() */
 #include <X11/Xlib.h>
```
Comment (psychon) [inline objects/key.c]: Some day (tm) I will also get rid of that one. But today is not that day...

Comment (Airblader) [inline objects/key.c]: One of the main reasons xcb doesn't have this yet, BTW, was the fact that xcb had no xrm equivalent. It should now be portable to XCB according to a discussion from 2008. :-)

**Comment by Elv13** [comment]
Thanks!

Welcome to the AwesomeWM contributors club ;) <sub>(yes, I know)</sub>

**Comment by Airblader** [comment]
@psychon helped us out over at i3 so many times, it's the least I can do. :-)

**Pull request merged**